templates/include/glean/labeled_group.1.schema.json

```diff
@@ -1,5 +1,6 @@
 "type": "object",
 "propertyNames": {
   "type": "string",
-  "pattern": "^[a-z_][a-z0-9_.]*$"
+  "pattern": "^[a-z_][a-z0-9_]{0,29}(\\.[a-z_][a-z0-9_]{0,29})*$",
+  "maxLength": 61
```

Comment:
In the glean-ac code, the max label length is 30.  We should probably make this match.

Reply:
Good catch!